code/modules/antagonists/abductor/machinery/camera.dm

```diff
@@ -59,16 +59,24 @@
 	return HAS_TRAIT(H, TRAIT_ABDUCTOR_SCIENTIST_TRAINING)
 
 /datum/action/innate/teleport_in
+///Is the amount of time required between uses
+	var/abductor_pad_cooldown = 8 SECONDS
+///Is used to compare to world.time in order to determine if the action should early return
+	var/on_cooldown
```

Comment:
`on_cooldown` sounds like a bool. Could you try something like `cooldown_time`, `next_use_time` or the likes?